mongo/content.md

```diff
@@ -37,7 +37,7 @@ See the [official docs](http://docs.mongodb.org/manual/) for infomation on using
 Just add the `--storageEngine` argument if you want to use the WiredTiger storage engine in MongoDB 3.0 and above without making a config file. Be sure to check the [docs](http://docs.mongodb.org/manual/release-notes/3.0-upgrade/#change-storage-engine-to-wiredtiger) on how to upgrade from older versions.
 
 ```console
-$ docker run --name some-mongo -d mongo --storageEngine=wiredTiger
+$ docker run --name some-mongo -d mongo mongod --storageEngine wiredTiger
```

Comment:
I'm still confused why this is necessary though, given the following block in the entrypoint script, which will see the leading `-` and prepend `mongod` automatically: https://github.com/docker-library/mongo/blob/c664ab653154aa4068a46f57ceeacd7c5cda1ae4/3.0/docker-entrypoint.sh#L4-L6